test/tracing/reactnativetracing.test.ts

```diff
@@ -295,9 +296,10 @@ describe('ReactNativeTracing', () => {
         const timeOriginMilliseconds = Date.now();
         const appStartTimeMilliseconds = timeOriginMilliseconds - 65000;
         const mockAppStartResponse: NativeAppStartResponse = {
-          isColdStart: false,
-          appStartTime: appStartTimeMilliseconds,
-          didFetchAppStart: false,
+          type: 'warm',
+          app_start_timestamp_ms: appStartTimeMilliseconds,
+          has_fetched: false,
```

Comment:
https://github.com/getsentry/sentry-react-native/pull/3855/files#diff-8fe4b2c9966b386116cb1a00d65b34b257134d0f598817b95e4052546df00bf8R320-R323

This test should be updated to use `has_fetched` instead of `didFetchAppStart`